arm-dns/2016-04-01/swagger/dns.json

```diff
@@ -0,0 +1,1202 @@
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "zoneName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the zone without a terminating dot."
+          },
+          {
+            "name": "IfMatch",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "description": "Defines the If-Match condition. The delete operation will be performed only if the ETag of the zone on the server matches this value."
+          },
+          {
+            "name": "IfNoneMatch",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "description": "Defines the If-None-Match condition. The delete operation will be performed only if the ETag of the zone on the server does not match this value."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/ZoneDeleteResult"
+            }
+          },
+          "202": {
+            "description": ""
+          },
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/ZoneDeleteResult"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "get": {
+        "tags": [
+          "Zones"
+        ],
+        "operationId": "Zones_Get",
+        "description": "Gets a DNS zone.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "zoneName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the zone without a terminating dot."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/Zone"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/dnszones": {
+      "get": {
+        "tags": [
+          "Zones"
+        ],
+        "operationId": "Zones_ListZonesInResourceGroup",
```

Comment:
Zones_List~~Zones~~InResourceGroup. This gets repetitive.
client.zones.listInREsourceGroup() will look much better